{{cookiecutter.slug}}/backend/Dockerfile-postgenerate

```diff
@@ -0,0 +1,7 @@
+FROM python:3.9
+
+WORKDIR /usr/src/app/backend
+COPY requirements.in .
+RUN pip install -U pip pip-tools
+
+CMD pip-compile
```

Comment:
There is one problem with `pip-comile`ing on the image: The resulting `requirements.txt` won't be present on the host machine, and hence cannot be checked into source control. I think some client-side shell script would be needed (`docker cp {backend-container-name}:/usr/src/app/backend/requirements.txt backend`) during post generation.

Reply:
That's also an option.